src/main/kotlin/nl/connectplay/scoreplay/data/DatabaseSessionRepository.kt

```diff
@@ -51,18 +53,19 @@ class DatabaseSessionRepository(private val database: Database) : SessionReposit
         }
     }
 
-    override suspend fun getSessionByIdAsync(sessionId: UUID): SessionDto? = coroutineScope {
+    override suspend fun getSessionByIdAsync(sessionId: UUID, userId: Int): SessionDto? = coroutineScope {
         async {
             database.connection?.use { connection ->
                 val sql = """
-                    SELECT s.session_id, s.game_id, s.host_user_id, s.session_visibility, p.picture_url as end_of_session_picture
+                    SELECT s.session_id, s.game_id, s.host_user_id, s.start_time, s.end_time,s.session_visibility, p.picture_url as end_of_session_picture
                     FROM sessions as s
                     LEFT JOIN pictures AS p ON s.end_of_session_picture_id = p.picture_id
-                    WHERE session_id = ?
+                    WHERE session_id = ? AND host_user_id = ?
                 """.trimIndent()
 
                 val stmt = connection.prepareStatement(sql)
                 stmt.setObject(1, sessionId)
+                stmt.setInt(1, userId)
```

Comment:
```suggestion
                stmt.setString(1, sessionId.toString())
                stmt.setInt(2, userId)
```
vgm was ik dit maar ja